### PR TITLE
moved user to top App, ammended the interests selection/deselection a…

### PR DESCRIPTION
**Pull request by nabilwardeh**
…nd the redirection

### CODE REVIEW

File: api/src/config-old/config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports.Server = require('./server');
```
Comment (kkarimi) [inline api/src/config-old/config.js]: remove `config-old`

File: api/test/seed.edited.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
+{
```
Comment (kkarimi) [inline api/test/seed.edited.json]: I think this is not neccesary for `json`

File: web/src/components/views/LandingPage.js [hunk 3]
```diff
@@ -29,48 +19,25 @@ export default class LandingPage extends Component {
...
-      );
-    }
     return (
       <Container id="btnFacebook">
         <Row>
           <Col className="sm-4" />
           <Col className="sm-4">
             <FacebookLogin
-              appId="133088487346292"
+              appId="138462666798513"
```
Comment (kkarimi) [inline web/src/components/views/LandingPage.js]: See if you can make your account admin / find a way around it

Comment (kkarimi) [reply web/src/components/views/LandingPage.js]: https://stackoverflow.com/questions/7225384/how-can-an-administrator-add-a-developer-to-an-app

Comment (nabilwardeh) [reply web/src/components/views/LandingPage.js]: We have agreed to use this app id

**Comment by kkarimi** [comment]
please rebase from `master` to resolve conflicts

**Pull request closed without merging**